src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="products" element={<Products />} />
```

Comment:
Route Path works with out "/", but I think it is a good practice to use forward slash.